packages/storebuilder/src/utils/handleTelemetryRequest.ts

```diff
@@ -4,6 +4,7 @@ interface TelemetryDataPayloadInterface {
 	_wpnonce: string;
 	action: string;
 	plugin_slug?: string;
```

Comment:
Is `plugin_slug` still needed or will it only ever use `pluginSlugs`?

Reply:
it's needed for the payment wizards. so shipping uses pluginSlugs, each payment wizard uses plugin_slug